render/wgpu/src/filters/glow.rs

```diff
@@ -15,9 +15,9 @@ use wgpu::util::DeviceExt;
 struct GlowUniform {
     color: [f32; 4],
     strength: f32,
-    inner: u32,    // a wasteful bool, but we need to be aligned anyway
-    knockout: u32, // a wasteful bool, but we need to be aligned anyway
-    _pad: f32,
+    inner: u32,            // a wasteful bool, but we need to be aligned anyway
+    knockout: u32,         // a wasteful bool, but we need to be aligned anyway
+    composite_source: u32, // undocumented flash feature, another bool
```

Comment:
Could this use a single `u32` with the three bools stored in the lower bits?

Reply:
It could, but as the comment says we need to stay aligned. 16 bytes for webgl, which means the fields won't go anywhere and we'd just be cramming out data for no reason.